tests/cypress/integration/directives/x-model.spec.js

```diff
@@ -279,3 +279,34 @@ test(
     }
 );
 
+test(
+    'x-model with fill on undefined custom element',
+    html`
+        <script>
+            class CustomElement extends HTMLElement {
+                connectedCallback() {
+                    this.innerText = "Loaded"
+                }
+
+                get value() {
+                    return "This is the internal value"
+                }
+
+            }
+
+            setTimeout(() => {
+                customElements.define("custom-element", CustomElement)
+            }, 300)
```

Comment:
I'm don't know Cypress very well. So I appreciate feedback on how to delay the definition of the custom element.